src/main/java/com/fulcrumgenomics/util/DemuxFastqs.java

```diff
@@ -677,14 +697,24 @@ void convertQuality(final byte[] quals, final FastqQualityFormat version) {
             return sampleBarcodeQualities;
         }
 
-        private String getMolecularBarcode(final List<FastqRecord>  records) {
-            final byte[][] molecularBarcode = new byte[records.size()][];
+        private byte[][] getMolecularBarcodes(final List<FastqRecord> records) {
+            final byte[][] molecularBarcodes = new byte[records.size()][];
```

Comment:
I was using `String barcodeSeqsToString`.  I suppose I can update it to use the `List<String>` version even (now that I know it exists.